bigbluebutton-html5/private/config/settings.yml

```diff
@@ -97,6 +97,7 @@ public:
     enableScreensharing: false
     enableVideo: true
     enableVideoStats: false
+    disableVideoMenu: false
```

Comment:
I'd encourage to rename the new field to be `enable...` to match the pattern of the other flags around it